### DISCUSSION TITLE
站點的SSL KEY 在面版上有多餘的換行字符

### DISCUSSION THREAD

**Discussion by WillWang2122** [1Panel-dev/1Panel#4969] [Q&A]
### 1Panel 版本

1.10.6-tls

### 问题描述

如題，有多餘的換行字符導至SSL 不可用... 要手動處理...

### 重现步骤

申請完證書後，在面版上看private KEY 和 public key

### 期待的正确结果

沒有多餘換行

### 附加信息
![asdf](https://github.com/1Panel-dev/1Panel/assets/168748301/db14b213-0850-494e-9a8d-4cbb3020e508)

_No response_

**Comment by wanghe-fit2cloud** [comment]
感谢反馈。
本地环境测试创建证书并绑定到网站，功能均正常，未能发现证书不可用的问题。

> 你可以先自行查看一下网站日志。
